data/ui_scripts/server_list/serverlist.lua

```diff
@@ -13,6 +13,7 @@ game:addlocalizedstring("SERVERLIST_SERVER_COUNT", "&&1 Servers")
 game:addlocalizedstring("SERVERLIST_ADD_TO_BLACKLIST", "Add to blacklist")
 game:addlocalizedstring("SERVERLIST_REMOVE_FROM_BLACKLIST", "Remove from blacklist")
 
+game:addlocalizedstring("LUI_MENU_PUBLIC", "Public")
```

Comment:
What about built-in 
`"MENU_SB_TOOLTIP_TAB_INTERNET_SERVERS"` _(Internet servers)_
`"MENU_SB_TOOLTIP_PUBLIC_SERVER"` _(Public server)_
or `"MENU_PUBLIC_PLAYLISTS"` _(Public Playlists)_
instead of custom _"Public"_?